src/status_im/contexts/communities/actions/share_community/style.cljs

```diff
@@ -0,0 +1,52 @@
+(ns status-im.contexts.communities.actions.share-community.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def header-container
+  {:padding-horizontal 20
+   :padding-vertical   12})
+
+(def scan-notice
+  {:color        colors/white-70-blur
+   :margin-top   16
+   :margin-left  :auto
+   :margin-right :auto})
+
+(defn community-name
+  [thumbnail]
+  {:margin-left (when thumbnail 8)})
+
+(def qr-code-wrapper
+  {:padding-horizontal 20
+   :margin-top         8})
+
+(def gradient-cover-padding 20)
+(def qr-code-padding 12)
+
+(defn qr-code-size
+  [total-width]
+  (- total-width (* gradient-cover-padding 2) (* qr-code-padding 2)))
+
+(defn gradient-cover-size
+  [total-width]
+  (- total-width (* gradient-cover-padding 2)))
+
+(defn gradient-cover-wrapper
+  [width]
+  {:width         (gradient-cover-size width)
+   :border-radius 16
+   :margin-left   20
+   :height        "100%"})
```

Comment:
could you elaborate why 100% is needed

Reply:
Because the current implementation of the gradient has a fixed value, And for this usage we want to make cover the entire wrapper.
https://github.com/status-im/status-mobile/blob/f708cdcf5075a26eb929585adb2f311e07b4d694/src/quo/components/gradient/gradient_cover/style.cljs#L3-L11

Reply:
I needed to override that